app/routes/items/new.js

```diff
@@ -73,10 +90,40 @@ export default AuthorizeRoute.extend({
         controller.set("length", null);
         controller.set("width", null);
         controller.set("height", null);
-        controller.set("selectedGrade", { name: "B", id: "B" });
-        controller.set("selectedCondition", { name: "Used", id: "U" });
+        controller.set("selectedGrade", {
+          name: "B",
+          id: "B"
+        });
+        controller.set("selectedCondition", {
+          name: "Used",
+          id: "U"
```

Comment:
Still don't understand how an id of `"U"`works

Reply:
It's an existing piece of code @patrixr, We need to check how this U is handled on the API side.

Reply:
I checked about that selectedCondition. It's not used anywhere in the `new.js` controller or `new.hbs` file. We can remove it.